### ISSUE TITLE
DCC page migration

### ISSUE THREAD

**Issue by gabrielaperezdelaosa** [cityofaustin/techstack#4731]
- [x] Create a department, contact, and locations in Joplin as necessary.
- [x] Create a new issue (with a target date) for each page that needs to be created in Joplin. Each of these 

issues should contain the following steps:

> - [ ] create a page that migrates [URL(s)] using current content types
> - [ ] make no additions to the text, you may delete redundant text
> - [ ] log content debt
>    - [ ] note common term usage in the word bank 'Monstro'
>    - [ ] create Github issues (tag as content debt)
>    - [ ] also record content debt in Joplin notes
> content debt includes: fact checking, reading level, length, accessibility, questions about the user or the necessity of this page, user research debt, service design debt, policy lab work, and new features to dev.
> - [ ] move to review and tag [reviewer]
>    - [ ] Reviewer proofreads the content
>    - [ ] If applicable, check implementation of fact checking
>    - [ ] Hit publish and copy URLs into the inventory
> If you can’t publish yet because you are waiting on the IA, you can still close the issue after the review is complete. 

Close this issue when
- [x] all pages are drafted and reviewed
